plantcv/plantcv/visualize/obj_size_ecdf.py

```diff
@@ -0,0 +1,47 @@
+# Plot Empirical Cumulative Distribution Function for Object Size
+
+import os
+import cv2
+import pandas as pd
+import numpy as np
+from plantcv.plantcv import params, fatal_error
```

Comment:
```suggestion
from plantcv.plantcv import params
```

Remove unused import